reactioncommerce/reaction
# Order dashboard search by orderId, customer info, order product info

Resolves #2353, #2354 and #2355 

## How To Test
- Create a few orders
- Search the orders using the Search Bar at the top with:
** OrderId
** Customer info (name, email, phone, card - last 4 digits)
** Product title, Variant title, Variant option title

- Confirm that all these search and any active filter takes effect on the results being displayed by the table.
- Confirm that clearing all filters and using search alone also works.



.
The only part of this pending is opening up the order summary action view if exactly one match/result is present in the table. Currently working on making Sortable not prevent the opening, but, I think reviews can start on this. I'll get the opening in before review completion.

**Update:**
There is a blocker to sliding out the table when there's only one matching item because the data fetching & rendering is happening within the table itself. Since we'll be moving the fetching #2768, that seem a better time to achieve this.
